### PR TITLE
Fix example image path

### PR DESCRIPTION
**Pull request by robb-romans**
Make the example path to local images more explicit.

### CODE REVIEW

File: contribute.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ order. Link to the appropriate graphic rather than duplicating it.
 To reference image files in GitHub, use the following markup:
 
     {% figure [caption:"<caption information>"] [class:"image bg-light border"] %}
-    ![]({{site.url}}<relative-path>){:width="100%"}
+    ![]({{site.url}}/assets/images/<relative-path>){:width="100%"}
```
Comment (nimbinatus) [inline contribute.md]: Remove the `/` after `{{site.url}}`:
```
![]({{site.url}}assets/images/<relative-path>){:width="100%"}
```

File: contribute.md [hunk 2]
```diff
@@ -65,13 +65,13 @@ order. Link to the appropriate graphic rather than duplicating it.
...
     {% endfigure %}
 
 For example, if you are working on the Modal page:
 
     {% figure [caption:"Diagram showing modal operation"] [class:"image bg-light border"] %}
-    ![]({{site.url}}components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
+    ![]({{site.url}}/assets/images/components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
```
Comment (nimbinatus) [inline contribute.md]: Same here:
```
![]({{site.url}}assets/images/components/content-areas/modals/modal-diagram-1.svg){:width="100%"}
```

File: contribute.md (hunk 1 above)
Comment (robb-romans) [reply contribute.md]: Based on my testing, it does not work without that slash.

File: contribute.md (hunk 2 above)
Comment (robb-romans) [reply contribute.md]: Based on my testing, it does not work without that slash.

File: contribute.md (hunk 1 above)
Comment (robb-romans) [reply contribute.md]: See https://github.com/rackerlabs/design-system/pull/265

Comment (nimbinatus) [reply contribute.md]: You're right; the CDN variable requires no slash, but the regular site URL variable does. Dismissing review.

File: contribute.md (hunk 2 above)
Comment (nimbinatus) [reply contribute.md]: Corrected

**Comment by robb-romans** [comment]
@nimbinatus thanks! if you're good with this, could you please merge?

**Comment by nimbinatus** [comment]
@robb-romans, done

**Pull request merged**